src/AddTodoForm.js

```diff
@@ -1,15 +1,34 @@
 import React from "react";
 
-function AddTodoForm() {
+
+function AddTodoForm(props) {
+    const handleChange = (event_item) => {
+        // console.log(event_item.target.value)
+        props.onAddTodo(event_item.target.value);
+    };
+
+    function handleAddTodo(events) {
```

Comment:
We only pass in one event at a time so let's use the singular word `event` as a parameter

Reply:
The word event wasn't working for me? Vs Code would cross it out. Is there something I need to do?

Reply:
VS code tends to do that. Ignore the crossout - `event` is the correct word to use in this sense.